examples/misc/test/effective_dart_test.dart

```diff
@@ -13,4 +13,43 @@ void main() {
     expect('abcd'.substring(1, 3), /**/ 'bc');
     // #enddocregion param-range
   });
+
+  test('toList()', () {
+    {
+      var iterable = [1, 2];
+      // #docregion list-from-1
+      var copy1 = iterable.toList();
+      var copy2 = new List.from(iterable);
+      // #enddocregion list-from-1
```

Comment:
FYI, I would have left this in the other file since there isn't anything to test (since the associated prose makes no claims that need to be tested).